### DISCUSSION TITLE
Github Action Disable

### DISCUSSION THREAD

**Discussion by tgxthings** [community/community#29391] [Actions]
Anyway to enable github action disabled by github

**Comment by MadhushaPrasad** [comment]
You can check this link to get some details about [Disabling and enabling a workflow](https://docs.github.com/en/actions/managing-workflow-runs/disabling-and-enabling-a-workflow)

**Reply by Wep-wep** [reply]
Pero porque deshabilitar?

**Comment by airtower-luna** [comment]
If it's been disabled *by GitHub* you'll have to [contact GitHub support](https://support.github.com/request) about it, us other users can't help there. And you'll probably have to fix whatever was the reason for disabling it. Good luck!
